internal/v2/s2av2.go

```diff
@@ -45,6 +45,14 @@ const (
 	s2aSecurityProtocol = "tls"
 )
 
+// S2AStream defines the operation for communicating with the S2A server over a bidirectional stream.
+type S2AStream interface {
```

Comment:
Per offline conversation, can we move this interface to its own package?

Reply:
Moved to a separate package!